modyn/config/schema/pipeline-schema.yaml

```diff
@@ -37,6 +37,61 @@ properties:
           Configuration dictionary that will be passed to the model on initialization.
     required:
       - id
+  model_storage:
+    type: object
+    properties:
+      full_model_strategy:
```

Comment:
Since the framework is based on full and incremental model strategies, I think that full model makes more sense.
However, I can change it if you want.